nilearn/glm/first_level/first_level.py

```diff
@@ -986,6 +986,7 @@ def first_level_from_bids(
     sub_labels : :obj:`list` of :obj:`str`, optional
         Specifies the subset of subject labels to model.
         If 'None', will model all subjects in the dataset.
+
```

Comment:
This is unrelated but came across a directive that did not render in the doc.

https://nilearn.github.io/stable/modules/generated/nilearn.glm.first_level.first_level_from_bids.html#nilearn.glm.first_level.first_level_from_bids

![image](https://github.com/nilearn/nilearn/assets/6961185/e4856baa-8b3a-4ecf-9df0-251425fba2ad)